app/javascript/widget/components/Banner.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div
+    v-if="showBannerMessage"
+    class="banner"
+    :class="`banner banner__${bannerType}`"
```

Comment:
We could omit the 'banner' class on line above this or the one here, 'banner--${bannerType}' would be more accurate syntax for bem.